site/no-nb/index.html

```diff
@@ -0,0 +1,669 @@
+                <br><br>
+                GS Collections har blitt presentert på en rekke
+                <a href="https://github.com/eclipse/eclipse-collections/wiki/Conference-talks-and-meetups">konferanser</a>,
+                inkludert JVM Summit i 2012 og JavaOne i 2014.
+                En sammenligning av ytelse mellom parallelle lazy implementasjoner i Java 8, Scala og GS Collections ble presentert på
+                <a href="http://www.infoq.com/presentations/java-streams-scala-parallel-collections">QCon New York</a> i 2014.
+                Videre ble <a href="https://github.com/eclipse/eclipse-collections/wiki/Articles">artikler</a> om GS Collections
+                (<a href="http://www.infoq.com/articles/GS-Collections-by-Example-1">Del 1</a> /
+                <a href="http://www.infoq.com/articles/GS-Collections-by-Example-2">Del 2</a>)
+                publisert på InfoQ.com.
+                Disse artikler viser noen av funksjonene i rammeverket gjennom eksempler, samt et intervju med skaperen av GS Collections.
+                <br><br>
+
+                Gjennom årene har rundt 40 utviklere fra det samme selskapet bidratt til rammeverket.
+                <br><br>
+
+                For å maksimere prosjektets potensiale som et åpen kildekode-prosjekt, ble GS Collections migrert til 
+                Eclipse Foundation og endret navn til Eclipse Collections i 2015. 
+                Nå er rammeverket helt åpent og aksepterer bidrag fra alle!
+            </p>
+        </div>
+    </div>
+    <!-- History  -->
+
+    <a class="anchor" name="refGuide"></a>
+    <div class="content-section-a">
+        <div class="container-fluid">
+            <h2 class="section-heading">Veiledning</h2>
+            <p class="lead">
+                En omfattende referansehåndbok for biblioteket er tilgjengelig på GitHub:
+                <a href="https://github.com/eclipse/eclipse-collections/blob/master/docs/guide.md#eclipse-collections-reference-guide">Reference Guide</a>.
+            </p>
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- Reference Guide  -->
+
+    <a class="anchor" name="sourceCode"></a>
+    <div class="content-section-b">
+        <div class="container-fluid">
+            <h2 class="section-heading">Kildekode</h2>
+            <p class="lead">
+                Kildekoden er tilgjengelig på GitHub: <a href="https://github.com/eclipse/eclipse-collections">Eclipse Collections</a>.
+            </p>
+        </div>
+        <!-- /.container -->
+    </div>
+    <!-- Source Code  -->
+
+    <a class="anchor" name="contribute"></a>
+    <div class="content-footer">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-4 col-sm-4">
+                    <h3>Bidra</h3>
+                    <p class="regular">
+                        Vi tar imot bidrag med glede!
```

Comment:
I'd say either "Vi tar med glede imot bidrag" or "Vi tar gjerne imot bidrag", perhaps preferably the first one, sounds a bit more natural.

Reply:
Agreed!